interview/holiday.py

```diff
@@ -9,4 +9,35 @@ def monthly_holiday(month: str) -> str:
 
     # TODO implement monthly_holiday function
 
-    return ''
+    if month == 'December':
+        return 'Hazels Birthday' 
```

Comment:
we needed to look at test_holiday.py to see what holidays this function expected!